Rubberduck.Refactorings/ExtractInterface/InterfaceMember.cs

```diff
@@ -56,7 +56,7 @@ public InterfaceMember(Declaration member)
             Identifier = member.IdentifierName;
             Type = member.AsTypeName;
             
-            GetMethodType();
+            MemberType = GetMethodType(Member.Context);
```

Comment:
should be `member.Context` to allow arbitrary reordering (even to before the assignment of `member` to `Member`)